### DISCUSSION TITLE
Finding neighbouring nodes in a mesh

### DISCUSSION THREAD

**Discussion by shuertaspenen** [simnibs/simnibs#372] [9. Results and analysis] [answered]
Hi everyone, 

I have found the node in which I have the maximum E-Magnitude in my tDCS simulation. Now, I need to check the neighboring nodes to see the values for those nodes. How can I determine which nodes are neighboring to a specific node? The mesh I have is the tDCS interpolated (middle_gm_interpolation). 

Thanks for your help,

**Comment by jduemose** [comment, marked as answer]
Hi, if you know the index of the node and you load the mesh into python, you can use [this](https://github.com/simnibs/simnibs/blob/d045e8ccda6b349cbec639563af32dac69e95909/simnibs/mesh_tools/mesh_io.py#L329) function. E.g.,
```
import simnibs

filename_mesh = "path to mesh file"
node_idx = 1 # or whatever

mesh = simnibs.mesh_tools.mesh_io.read(filename_mesh)
nn = mesh.elm.find_neighbouring_nodes(node_idx)
```
Ignore any -1's that you may find in `nn`. Please note that node indexing starts from 1 but your index may also be 1-indexed.
